Neos.Flow/Documentation/Quickstart/index.rst

```diff
@@ -265,10 +261,10 @@ action method like this one::
     /**
      * This action outputs a custom greeting
      *
-     * @param string $name your name
      * @return string custom greeting
```

Comment:
If removing the `@param`, the `@return` can go, too, no? Because it doesn't really add any value here.